spec/kokoro-run-all.sh

```diff
@@ -197,4 +196,7 @@ else
   done
 fi
 
+chmod +x $KOKORO_GFILE_DIR/linux_amd64/buildcop
+$KOKORO_GFILE_DIR/linux_amd64/buildcop --repo GoogleCloudPlatform/ruby-docs-samples
```

Comment:
I sent https://github.com/googleapis/repo-automation-bots/pull/414 so you can remove the `-repo` flag.